Source/WTF/wtf/PlatformHave.h

```diff
@@ -804,8 +804,8 @@
 #define HAVE_CONTACTS 1
 #endif
 
-#if PLATFORM(IOS) || PLATFORM(MAC)
-#define HAVE_DIGITAL_CREDENTIALS_UI 1
+#if !defined(HAVE_DIGITAL_CREDENTIALS_UI)
+#define HAVE_DIGITAL_CREDENTIALS_UI 0
 #endif
```

Comment:
These lines can be removed entirely.

Reply:
As can "Explanation of why this fixes the bug (OOPS!)."